### PR TITLE
Create wasm_stats.sql

### PR DESCRIPTION
**Pull request by rviscomi**
cc @RReverser

### CODE REVIEW

**Comment by rviscomi** [comment]
@tunetheweb the linter is complaining but I think it's wrong :)

Merging anyway.

**Comment by tunetheweb** [comment]
No please. It's an encoding error. Not a SQL error. You must have copy and pasted from somewhere weird.

File: sql/util/wasm_stats.sql [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+      (url)
+    JOIN (
+      SELECT
+        url AS page,
+        pageid,
+        _TABLE_SUFFIX
+      FROM
+        `httparchive.summary_pages.2021_09_01_*`)
+    USING
+      (_TABLE_SUFFIX, pageid)))
```
Comment (tunetheweb) [inline sql/util/wasm_stats.sql:36]: CREATE OR REPLACE TABLE `httparchive.almanac.wasm_stats`
PARTITION BY date
CLUSTER BY client AS
SELECT DATE('2021-09-01') AS date, * FROM (
  SELECT * EXCEPT (_TABLE_SUFFIX), _TABLE_SUFFIX AS client
  FROM (SELECT * EXCEPT (size) FROM `blink-httparchive-research.rreverser2.wasms`)
  JOIN `blink-httparchive-research.rreverser2.wasm_stats` USING (filename)
  JOIN `httparchive.summary_requests.2021_09_01_*` USING (url)
  JOIN (SELECT url AS page, pageid, _TABLE_SUFFIX FROM `httparchive.summary_pages.2021_09_01_*`) USING (_TABLE_SUFFIX, pageid)
)

Comment (tunetheweb) [inline sql/util/wasm_stats.sql:36]:
```suggestion
CREATE OR REPLACE TABLE `httparchive.almanac.wasm_stats`
PARTITION BY date
CLUSTER BY client AS
SELECT DATE('2021-09-01') AS date, * FROM (
  SELECT * EXCEPT (_TABLE_SUFFIX), _TABLE_SUFFIX AS client
  FROM (SELECT * EXCEPT (size) FROM `blink-httparchive-research.rreverser2.wasms`)
  JOIN `blink-httparchive-research.rreverser2.wasm_stats` USING (filename)
  JOIN `httparchive.summary_requests.2021_09_01_*` USING (url)
  JOIN (SELECT url AS page, pageid, _TABLE_SUFFIX FROM `httparchive.summary_pages.2021_09_01_*`) USING (_TABLE_SUFFIX, pageid)
)
```

**Review by tunetheweb — CHANGES_REQUESTED** [review]

**Comment by tunetheweb** [comment]
Now you can merge :-)

The linter is never wrong - trust the linter!

**Review by tunetheweb — APPROVED** [review]

**Comment by rviscomi** [comment]
I should have known better 😅 

Thanks for your help pinning that down!

**Pull request merged**